### DISCUSSION TITLE
Renovate pre-commit hook exceeds pre-commit.ci tier max size

### DISCUSSION THREAD

**Discussion by 5ouma** [renovatebot/pre-commit-hooks#2983] [Q&A] [answered]
pre-commit.ci limits file size to 250MiB.
Is it possible to reduce the size?
https://results.pre-commit.ci/run/github/849736738/1733269225.7CEAHAS_Tfmga8vtCVKyZQ

**Comment by viceice** [comment, marked as answer]
not possible
